src/shared/migrations/040-add-slot-groups-parent-group.ts

```diff
@@ -0,0 +1,16 @@
+import { DataTypes } from 'sequelize';
+
+/**
+ * Adds the requiredDLCs column to the MissionSlots table
+ */
+module.exports = {
+    up: async (queryInterface: any): Promise<void> => {
+        await queryInterface.addColumn('missionSlotGroups', 'parentGroup', {
+            type: DataTypes.UUID,
+            allowNull: true
+        });
```

Comment:
`parentGroup` in migration, `parentGroupUid` in model
Change migration to match `@Attribute` decorator in model and add `null` default value:

```
await queryInterface.addColumn('missionSlotGroups', 'parentGroupUid', {
    type: DataTypes.UUID,
    allowNull: true,
    defaultValue: null,
    references: {
        model: 'missionSlotGroups',
        key: 'uid'
    },
    onDelete: 'CASCADE',
    onUpdate: 'CASCADE'
});
```